nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/SplitText.java

```diff
@@ -326,6 +329,27 @@ public void process(InputStream in) throws IOException {
         return splitFlowFiles;
     }
 
+    /**
+     * Will merge the two files. This operation has different semantics then
+     * sessioin.merge. Also, it always expects 2 and no more then 2 flow files,
+     * so vararg signature primarily for simplification.
+     */
+    private FlowFile merge(ProcessSession session, FlowFile... flowFiles) {
```

Comment:
So, in the context of this specific processor what you're saying is true - _prepend header_. But, the operation itself has nothing to do with either _headers_ or _prepend_.  It simply merges two files together in the order they have been supplied and eventually should probably be considered as candidate for some utility static operation. Anyway, I'll keep the name as is, but will update the javadoc.